.github/workflows/create-release.yml

```diff
@@ -0,0 +1,39 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+name: Create Release
+
+jobs:
+  build:
+    name: Upload Release Asset
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Prepare archive
+        run: |
+          ./foo.sh
```

Comment:
This correct?

Reply:
Most definitely not, but a leftover from testing. Fixed now.